pb_teleop_twist_joy:
  ros__parameters:
    use_sim_time: false
    robot_base_frame: gimbal_yaw
    control_mode: manual_control  # Option: auto_control, manual_control

    require_enable_button: true
    enable_button: 4              # L1 shoulder button
    enable_turbo_button: 5        # R1 shoulder button

    axis_chassis:
      x: 1                        # Left thumb stick vertical
      y: 0                        # Left thumb stick horizontal
      yaw: 6                      # button_left and button_right
    scale_chassis:
      x: 2.5
      y: 2.5
      yaw: 3.0
    scale_chassis_turbo:
      x: 4.0
      y: 4.0
      yaw: 6.0

    axis_gimbal:
      roll: -1                    # Disable
      pitch: 4                    # Right thumb stick vertical
      yaw: 3                      # Right thumb stick horizontal
      shoot: 7
    scale_gimbal:
      roll: 0.0
      pitch: -1.0
      yaw: 2.5
      shoot: 1.0
    scale_gimbal_turbo:
      roll: 0.0
      pitch: -1.5
      yaw: 3.5
      shoot: 1.0
